packages/storage/amplify_storage_s3/lib/src/S3PrefixResolver/AmplifyStorageS3PrefixResolver.dart

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Folders and files should be named using snake_case per Dart conventions: https://dart.dev/guides/language/effective-dart/style#do-name-libraries-and-source-files-using-lowercase_with_underscores

`s3_prefix_resolver/amplify_storage_s3_prefix_resolver.dart`

Reply:
shoot my bad thanks for catching that.